### ISSUE TITLE
BASE_URL changed to .com

### ISSUE THREAD

**Issue by smaira** [coder-alpha/Einthusan.bundle#30]
Einthusan.tv has been down for a few days so I went into the bundle and replaced "einthusan.tv" with "einthusan.com" in all source files. The movie list starts to appear again but the proxy doesn't work anymore. I am on OSx and it worked with just Alternate TLS a few days back, then stopped working (probably because .tv site went down) and now none of the settings work anymore. 

Any thoughts?

**Comment by sonybansi** [comment]
same here

**Comment by gsaxam** [comment]
I have the same issue, I tried turning off the alternate SSL and also tried through the proxy.

**Comment by veeruns** [comment]
a little bit of sed and some checking log later, works on my Linux server.

**Comment by smaira** [comment]
How did you make it work?

**Comment by gsaxam** [comment]
Hey @veeruns can you please create a new branch and post your changes there if possible? what parts did you replace with sed? modifying the python code is easy but testing the plex plugin takes a few steps every since time. Thanks!

**Comment by veeruns** [comment]
@gsaxam changed it in  Contents/Code/__init__.py Contents/Code/common.py Contents/Libraries/Shared/slimerjs/ Contents/Libraries/Shared/slimerjs/__init__.py Contents/Services/URL/CusEinthusan/ServiceCode.pys Contents/Code/einthusan.py

Had to change vidpath = furl.split('.tv/')[1] too.

Probably it was beginners luck, changing the python code worked for me. Sorry to leave you guys in the lurch,  cannot create a pull request at this time.

**Comment by kabindra** [comment]
#31 fix has been sent for a pull request.

**Comment by jbl0ggs** [comment]
> @gsaxam changed it in Contents/Code/**init**.py Contents/Code/common.py Contents/Libraries/Shared/slimerjs/ Contents/Libraries/Shared/slimerjs/**init**.py Contents/Services/URL/CusEinthusan/ServiceCode.pys Contents/Code/einthusan.py
> 
> Had to change vidpath = furl.split('.tv/')[1] too.
> 
> Probably it was beginners luck, changing the python code worked for me. Sorry to leave you guys in the lurch, cannot create a pull request at this time.

I'm sorry, I'm still lost. What exactly did you change in those files? Also, is there a special way to edit those files or can one just use NotePad?

Thanks for your help!

**Comment by smaira** [comment]
I made the changes using a text editor. The titles started loading but
video still doesn't stream. I'm using SlimerJS and have the correct path
for Firefox defined. I also have the proxy setting enabled.

**Comment by jbl0ggs** [comment]
> I made the changes using a text editor. The titles started loading but video still doesn't stream. I'm using SlimerJS and have the correct path for Firefox defined. I also have the proxy setting enabled.
> […](#)

Found a working bundle posted at the following link. Had to also disable Proxy.

https://forums.plex.tv/t/rel-einthusan-channel-indian-movies/104742/304

**Comment by kabindra** [comment]
Did you check #31. It already fixed and send for pull request.

**Comment by jbl0ggs** [comment]
> Did you check #31. It already fixed and send for pull request.

Yes, but not being familiar with the how bundles are setup, it was not really clear for me on EXACTLY what had to be changed. There was mention of changing from .TV to .COM and then talk about .CA and so it just did not make sense to me. Was just easier for me to download a bundle with all the changes made by someone who knew what they were doing.

**Comment by jbl0ggs** [comment]
I know this is for reporting issues for Einthusan bundle, but since the G2G.fm section seems to be abandoned, I thought I would ask here about it in case someone here is able to help.

I just wanted to report that the g2g.fm plex plugin has stopped working for me. Movies only appear under the “Latest Video” category. When clicking on the video, the message I get is Video not found.

Is there any resolution to this issue?
